From 3bc0cddabba890c7ce89beb24ba7497e5c94c76b Mon Sep 17 00:00:00 2001
From: Andreas Eliasson <andreas.eliasson@qt.io>
Date: Thu, 16 Oct 2025 13:03:34 +0200
Subject: [PATCH 229/239] Doc: Fix qml lint binding warnings

These snippets show up in the Qt Quick States documentation.

Fixes: QTBUG-140487
Change-Id: I897865e16a356c1c561715adc4daa97a4bd3dd67
Reviewed-by: Mitch Curtis <mitch.curtis@qt.io>
(cherry picked from commit be4c7d13729dbc6c7dcd2e42ae25a57f7a1b30f6)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/quick/doc/snippets/qml/states.qml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/quick/doc/snippets/qml/states.qml b/src/quick/doc/snippets/qml/states.qml
index 426ce3be4f..d412eb53f8 100644
--- a/src/quick/doc/snippets/qml/states.qml
+++ b/src/quick/doc/snippets/qml/states.qml
@@ -25,13 +25,13 @@ Rectangle {
     states: [
         State {
             name: "NORMAL"
-            PropertyChanges { target: signal; color: "green"}
-            PropertyChanges { target: flag; state: "FLAG_DOWN"}
+            PropertyChanges { signal.color: "green"}
+            PropertyChanges { flag.state: "FLAG_DOWN"}
         },
         State {
             name: "CRITICAL"
-            PropertyChanges { target: signal; color: "red"}
-            PropertyChanges { target: flag; state: "FLAG_UP"}
+            PropertyChanges { signal.color: "red"}
+            PropertyChanges { flag.state: "FLAG_UP"}
         }
     ]
 }
@@ -64,7 +64,7 @@ Rectangle {
     states: State {
                 name: "RINGING"
                 when: (signal.state == "CRITICAL")
-                PropertyChanges {target: speaker; play: "RING!"}
+                PropertyChanges {speaker.play: "RING!"}
             }
 }
 //![when property]
-- 
2.51.2

